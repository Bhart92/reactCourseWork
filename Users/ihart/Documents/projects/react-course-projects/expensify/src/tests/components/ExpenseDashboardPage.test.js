import React from 'react';
import { shallow } from  'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import NotFoundPage from '../../components/NotFoundPage';


test('Should render the Expense dashbaord page', () => {
	const wrapper = shallow(<ExpenseDashboardPage />);
	expect(wrapper).toMatchSnapshot();
});

test('Should render the not found page', () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(wrapper).toMatchSnapshot();

});
