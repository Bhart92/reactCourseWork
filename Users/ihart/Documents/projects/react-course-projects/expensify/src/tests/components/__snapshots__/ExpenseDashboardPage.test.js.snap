// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Expense dashbaord page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;

exports[`Should render the not found page 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
